src/sidebar/services/annotations.ts

```diff
@@ -53,11 +57,16 @@ export class AnnotationsService {
   }
 
   /**
-   * Extend new annotation objects with defaults and permissions.
+   * Create a new {@link Annotation} object from a set of field values.
+   *
+   * All fields not set in `annotationData` will be populated with default
+   * values.
    */
-  private _initialize(
-    annotationData: Omit<AnnotationData, '$tag'>,
```

Comment:
This type was previously inaccurate, as it reflected the type of annotation object coming back from the server, in which certain fields are always present (eg. `created`, `updated`) but this method is actually used for initializing newly created annotations locally, in which the various fields are not necessarily present.

This method was already used when creating new annotations locally, I have exposed it for use in initializing imported annotations.